core/components/blocks/Microcart/Microcart.js

```diff
@@ -20,11 +20,6 @@ export default {
     VueOfflineMixin
   ],
   props: {
-    product: {
-      type: Object,
```

Comment:
@filrak I removed it. I see we don't use it anywhere. It was added at beginning -> https://github.com/DivanteLtd/vue-storefront/commit/aa0e57abd0924fb5f0f6e365deb2f220fe41c002#diff-cb47176996933f7293e2e419bbfdca34R8